network-store-server/src/main/resources/db/changelog/changesets/changelog_20250415T064500Z.xml

```diff
@@ -0,0 +1,62 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd" logicalFilePath="db/changelog/changesets/changelog_2022-09-26T06:45:00Z.xml">
+    <changeSet author="lesoteti" id="1664174319767-1">
+        <createTable tableName="area">
+            <column name="networkuuid" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="area_pkey"/>
+            </column>
+            <column name="variantnum" type="INTEGER">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="area_pkey"/>
+            </column>
+            <column name="id" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="area_pkey"/>
+            </column>
+            <column name="name" type="VARCHAR(255)"/>
+            <column name="areatype" type="VARCHAR(255)"/>
+            <column name="voltagelevelids" type="TEXT"/>
+            <column name="interchangetarget" type="FLOAT8"/>
+            <column name="fictitious" type="BOOLEAN"/>
+            <column name="properties" type="TEXT"/>
+            <column name="aliaseswithouttype" type="TEXT"/>
+            <column name="aliasbytype" type="TEXT"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="lesoteti" id="1664174319767-2">
+        <createTable tableName="areaboundary">
+            <column name="areaid" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="networkuuid" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="variantnum" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="boundarydanglinglineid" type="VARCHAR(255)"/>
+            <column name="terminalconnectableid" type="VARCHAR(255)"/>
+            <column name="terminalside" type="VARCHAR(50)"/>
+            <column name="ac" type="BOOLEAN"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="lesoteti" id="1664174319767-3">
+        <createIndex indexName="areaboundary_networkuuid_variantnum_substationid_idx" tableName="areaboundary">
+            <column name="networkuuid"/>
+            <column name="variantnum"/>
+            <column name="areaid"/>
+        </createIndex>
+    </changeSet>
+    <changeSet id="lesoteti" author="1664174319767-4">
+        <addColumn tableName="battery">
```

Comment:
This is the old style for extensions storage. Should be in extension table. But that way, it's homogeneous with GeneratorShortcircuit...

Reply:
For now we let it in battery's table we will move both extension later